setup.py

```diff
@@ -16,6 +16,8 @@
     classifiers=[
         'Framework :: Plone',
         'Framework :: Plone :: 4.3',
+        'Framework :: Plone :: 5.0',
```

Comment:
I'd drop 5.0, since we are not officially supporting it (=testing it).

Reply:
Mh, but there is a `test-plone-5.0.x.cfg` since the beginning of the existence of this package, see https://github.com/4teamwork/ftw.slacker/blob/e840eb22d6a0c4e8e6ce1ece9ecc0613ab7ceaa0/test-plone-5.0.x.cfg

I have moved the update of the classifiers to its own commit to make this more clear.

Reply:
On the other hand, I could just remove the `test-plone-5.0.x.cfg` ;-)

Reply:
Ah, I wasn't aware of that 🤔 
I'd probably also remove Plone 5.0 and go with 5.1, because 5.0 will break at some point (wasn't as stable as 5.1 dependency-wise).

Reply:
Ok, I have deleted `test-plone-5.0.x.cfg`.